zebra-network/Cargo.toml

```diff
@@ -2,7 +2,9 @@
 name = "zebra-network"
 version = "1.0.0-beta.25"
 authors = ["Zcash Foundation <zebra@zfnd.org>"]
+description = "Networking code for Zebra"
 license = "MIT OR Apache-2.0"
+repository = "https://github.com/ZcashFoundation/zebra"
 edition = "2021"
 
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
```

Comment:
The Zebra dependencies in this file need versions added.

Reply:
Looks like the dev-dependencies do not need versions

Reply:
They aren't required to publish, but they do enable some useful Rust ecosystem tests, so I opened a ticket:
https://github.com/ZcashFoundation/zebra/issues/6924